go/store/prolly/tuple_map.go

```diff
@@ -246,6 +246,12 @@ func (m Map) Get(ctx context.Context, key val.Tuple, cb tree.KeyValueFn[val.Tupl
 	return m.tuples.Get(ctx, key, cb)
 }
 
+// GetPrefix searches for the key-value pair keyed by |key| and passes the results to the callback.
```

Comment:
We should mention that this method is only valid for indexes where `prefDesc` corresponds to a set of unique columns